.github/workflows/post-merge-dynamo-to-dev.yml

```diff
@@ -36,9 +37,7 @@ jobs:
           aws-region: ${{ env.AWS_REGION }}
 
       - name: SAM Validate
-        run: sam validate --region ${{ env.AWS_REGION }} -t 
-          working-directory: infra-l2-dynamo/**
-          template-file: template.yaml
+        run: sam validate --region ${{ env.AWS_REGION }} -t
```

Comment:
this will need `template.yaml` added inline after the -t.
it's a run command so wont pick up the default var.
i'd recommend removing